src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -55,6 +58,7 @@ public double getVelocity(){
 
   @Override
   public void periodic() {
+    
```

Comment:
Please add the current velocity of the shooter to the dashboard, and the command velocity too.